Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -208,4 +208,17 @@ class BrowserToolbar: Toolbar, BrowserToolbarProtocol {
         stopReloadButton.enabled = isWebPage
         shareButton.enabled = isWebPage
     }
+
+    override func drawRect(rect: CGRect) {
+        let context = UIGraphicsGetCurrentContext()
+        drawLine(context, start: CGPoint(x: 0, y: 0), end: CGPoint(x: frame.width, y: 0))
```

Comment:
Ya, I wondered the same, but just followed https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Widgets/Toolbar.swift#L26 as an example.